plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -108,7 +283,7 @@ def gas_station_gas_price_strategy(  # pylint: disable=redefined-outer-name,unus
             raise ValueError(f"Invalid return value for `{gas_price_strategy}`!")
         gwei_result = result / 10  # adjustment (see api documentation)
         wei_result = web3.toWei(gwei_result, "gwei")
-        return wei_result
+        return {"gasPrice": wei_result}
```

Comment:
Then you need to change it for the `rpc_gas_price_strategy` above too, otherwise there are varying return types.